src/System.Management.Automation/help/CommandHelpProvider.cs

```diff
@@ -542,6 +548,13 @@ private string GetHelpFile(string helpFile, CmdletInfo cmdletInfo)
         /// <returns></returns>
         private string FindHelpFile(CmdletInfo cmdletInfo)
         {
+            if (InternalTestHooks.BypassOnlineHelpRetrieval)
+            {
+                // By returning null, we force get-help to generate a metadata driven help object,
+                // which includes a helpUri that points to the fwlink defined in the cmdlet code.
```

Comment:
I'm asking this because I don't see this test hook is used for this purpose in the tests from this PR.